framework/h5py_interface_creator.py

```diff
@@ -239,8 +254,11 @@ def addGroup(self,rlz):
       self.__addGroupRootLevel(groupName,rlz)
       self.firstRootGroup = True
       self.type = 'MC'
-    self.h5FileW.attrs['allGroupPaths'] = _dumps(self.allGroupPaths)
-    self.h5FileW.attrs['allGroupEnds'] = _dumps(self.allGroupEnds)
+
+    self.h5FileW["allGroupPaths"].resize((len(self.allGroupPaths),))
+    self.h5FileW["allGroupEnds"].resize( (len(self.allGroupEnds),) )
+    self.h5FileW["allGroupPaths"][...] = self.allGroupPaths
+    self.h5FileW["allGroupEnds"][...] = self.allGroupEnds
```

Comment:
I dont recall seeing the "[...]" notation before

Reply:
```Regular assignment in Python simply changes a reference in the local or global variable dictionary instead of modifying an existing variable in place. This means that simply assigning to x will not place the value into the element of the array, but rather switch x from being an array element reference to being a reference to the value you assigned. To actually modify the element of the array, x should be indexed with the ellipsis.```